src/aliased_buffer.h

```diff
@@ -3,9 +3,9 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-#include <cinttypes>
-#include "util.h"
-#include "v8.h"
+# include <cinttypes>
+# include "util.h"
+# include "v8.h"
```

Comment:
I'm thinking that `#include` statements should definitely be explicitly excluded from the indent rule.

Reply:
I don’t think `#include` statements should get special handling. Rather, the standard header guards above should not count towards the indent depth, imo.

Reply:
That works also :-)

Reply:
I've not see a space after the preprocessor directive # before. I'd prefer not having them but I can live with that if others think this is alright.

Reply:
I implemented @addaleax 's suggestion there